bin/lang-js/src/function.ts

```diff
@@ -91,12 +91,25 @@ export interface OutputLine {
   message: string;
 }
 
+// This is the default timeout, configured in minutes.
+const defaultTimeout = 60; // TODO(nick): make this configurable when deploying SI
+
 export async function executeFunction(kind: FunctionKind, request: Request) {
   // Run Before Functions
   const ctx = ctxFromRequest(request);
 
+  const timeout = request.timeout ?? defaultTimeout;
+  console.log(JSON.stringify({
+    protocol: "output",
+    executionId: ctx.executionId,
+    stream: "output",
+    level: "debug",
+    group: "log",
+    message: `Timeout in minutes: ${timeout}`,
+  }));
```

Comment:
@johnrwatson @sprutton1 @vbustamante new log for timeout detection at the "debug" level